src/react/databrowser/objects/ObjectDelete.jsx

```diff
@@ -17,18 +18,18 @@ const Files = styled.div`
     height: 250px;
     width: 500px;
     overflow-y: scroll;
-    margin: 10px 0px;
+    margin: ${padding.small} 0px;
     border: 1px solid ${props => props.theme.brand.borderLight};
 `;
 
 const Description = styled.div`
-    font-size: 14px;
-    margin-top: 2px;
+    font-size: ${fontSize.base};
```

Comment:
Do we need to still provide default/base font-size?
If we set a default font-size, should this component inherit from it? Same logic can be apply for the following `font-size: 14px;`.